tests/test_scheduler.py

```diff
@@ -561,8 +611,9 @@ def test_full_loop_no_noise(self):
             residual = model(sample, t)
             sample = scheduler.step_plms(residual, i, sample)["prev_sample"]
 
-        result_sum = torch.sum(torch.abs(sample))
-        result_mean = torch.mean(torch.abs(sample))
+        import ipdb; pdb.set_trace()
```

Comment:
We may need to remove this :)

Reply:
Oh yeah, I posted the code mostly to share in case anyone wanted to try, but it was a little rude of my to not polish this off.

Reply:
No worries!